components/01-globals/_fonts/_fonts.scss

```diff
@@ -0,0 +1,27 @@
+@font-face {
+    font-family: 'Montserrat';
+    src:
+    url('/components/raw/fonts/montserrat-regular.woff2') format('woff2'),
+    url('/components/raw/fonts/montserrat-regular.woff') format('woff'),
+    url('/components/raw/fonts/montserrat-regular.svg') format('svg');
```

Comment:
Unfortunately current linter configuration is not allowing to do that. It expects 4 spaces indentation. When I put more...
![image](https://user-images.githubusercontent.com/12099511/31654206-92e00fca-b325-11e7-89c1-7f0a9b8db3d2.png)
With 4 spaces : 
![image](https://user-images.githubusercontent.com/12099511/31654270-c5f4f2ea-b325-11e7-97fc-e2a826b01f79.png)
I have used this formatting because I believe that this:
![image](https://user-images.githubusercontent.com/12099511/31654354-22724fd6-b326-11e7-877e-ad3a476e328e.png)
Is more readable than this:
![image](https://user-images.githubusercontent.com/12099511/31654383-331657f6-b326-11e7-9af0-0892e0301efd.png)
What do you think Anna ?

Reply:
I think you're right :) Thank you for this really visually explanation :) 
It's weird a bit for me but if we should format text so... let's leave it :)